File: pset6/cash.py
import math
from cs50 import get_float

while True:
    n = get_float("Change: ")
    if n > 0:
        break
# to change the dollars into cents:
i = round(n * 100)
coin_amount = 0
# when over a quarter
while i >= 25:
    i -= 25
    coin_amount += 1
# when over a dime
while i >= 10:
    i -= 10
    coin_amount += 1
# when over a nickel
while i >= 5:
    i -= 5
    coin_amount += 1
# when over a penny
while i >= 1:
    i -= 1
    coin_amount += 1

print(coin_amount)

File: pset1/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float n;
    do
    {
        n = get_float("Change: ");
    }
    while (n < 0);

//To change the dollars into cents:
    int i = round(n * 100);
  
    int coint_ammount = 0;
//When over a quarter, subtract and add 1 to variable coint_ammount
    while (i >= 25)
    {
        i -= 25,
        coint_ammount++;
    }
//When over a dime, subtract and add 1 to variable coint_ammount
    while (i >= 10)
    {
        i -= 10,
        coint_ammount++;
    }
//When over a nickel, subtract and add 1 to variable coint_ammount
    while (i >= 5)
    {
        i -= 5,
        coint_ammount++;
    }
//When over a penny, subtract and add 1 to variable coint_ammount
    while (i >= 1)
    {
        i -= 1,
        coint_ammount++;
    }
    printf("%i\n", coint_ammount);    
}
